sdks/python/setup.py

```diff
@@ -185,7 +185,8 @@ def run(self):
     author_email=PACKAGE_EMAIL,
     packages=setuptools.find_packages(),
     package_data={'apache_beam': [
-        '*/*.pyx', '*/*/*.pyx', '*/*.pxd', '*/*/*.pxd', 'testing/data/*.yaml']},
+        '*/*.pyx', '*/*/*.pyx', '*/*.pxd', '*/*/*.pxd',
+        'portability/api/*.yaml']},
```

Comment:
We still need to keep `'testing/data/*.yaml'` in addition to the new location because there is also another yaml file in the testing location (i.e. https://github.com/apache/beam/blob/master/sdks/python/apache_beam/testing/data/trigger_transcripts.yaml)